packages/wonder-blocks-tokens/src/tokens/semantic-color.ts

```diff
@@ -1,18 +1,87 @@
 import {color} from "./color";
 
+const border = {
+    primary: color.fadedOffBlack16,
+    subtle: color.fadedOffBlack8,
+    strong: color.fadedOffBlack50,
+    inverse: color.white,
+};
+
 export const semanticColor = {
     /**
      * For buttons, links, and controls to communicate the presence and meaning
      * of interaction.
      */
     action: {
-        primary: {
-            default: color.blue,
-            active: color.activeBlue,
+        // For primary actions
+        progressive: {
```

Comment:
For my own understanding, what was the reason for using the `progressive`/`progressiveInverse` terminology instead of `primary`/`secondary`? 

And should we start using this terminology for component `kind` variants too?